packages-exp/auth-exp/src/platform_browser/messagechannel/sender.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  SenderRequest,
+  EventType,
+  ReceiverMessageEvent,
+  MessageError,
+  SenderMessageEvent,
+  Status,
+  ReceiverMessageResponse,
+  ReceiverResponse,
+  TimeoutDuration
+} from './index';
+
+interface MessageHandler {
+  messageChannel: MessageChannel;
+  onMessage: EventListenerOrEventListenerObject;
+}
+
+function generateEventId(prefix: string = '', digits: number = 20): string {
```

Comment:
The types of `prefix` and `digits` can be inferred from the default value you provide; remove the explicit typings.
go/tsstyle#type-inference

Reply:
done